time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -408,7 +408,7 @@ def find_running(self):
         time_entry = self.repository.find_running(
             event_ctx.tenant_id, event_ctx.user_id
         )
-        self.stop_time_entry_if_was_left_running(time_entry)
+        # self.stop_time_entry_if_was_left_running(time_entry)
```

Comment:
Can we add little TODO here to add some context to this comment. Something like "Refactor automatic clock-out to consider local timezones". By using the word "refactor" I intend to include the intention to either remove the whole logic around this or extend it taking into account local timezones.